File: src/main/java/org/adoptopenjdk/jheappo/heapdump/AllocSites.java
package org.adoptopenjdk.jheappo.heapdump;

/*
 * Copyright (c) 2018 Kirk Pepperdine.
 * Licensed under https://github.com/AdoptOpenJDK/jheappo/blob/master/LICENSE
 * Instructions: https://github.com/AdoptOpenJDK/jheappo/wiki
 */


import org.adoptopenjdk.jheappo.io.HeapDumpBuffer;

public class AllocSites extends HeapDumpBuffer {

    public final static int TAG = 0x06;

    public AllocSites(byte[] body) {
        super(body);
    }
}

File: src/main/java/org/adoptopenjdk/jheappo/heapdump/AllocSites.kt
package org.adoptopenjdk.jheappo.heapdump

/*
 * Copyright (c) 2018 Kirk Pepperdine.
 * Licensed under https://github.com/AdoptOpenJDK/jheappo/blob/master/LICENSE
 * Instructions: https://github.com/AdoptOpenJDK/jheappo/wiki
 */


import org.adoptopenjdk.jheappo.io.HeapProfileRecord

class AllocSites(body: EncodedChunk) : HeapProfileRecord() {
    companion object {
        const val TAG = 0x06
    }
}
